### PR TITLE
feature/WAL-451

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

File: src/screens/SSICredentialsOverviewScreen/index.tsx [hunk 1]
```diff
@@ -72,24 +78,39 @@ class SSICredentialsOverviewScreen extends PureComponent<IProps, IState> {
...
+        title={itemInfo.item.title}
+        issuer={itemInfo.item.issuer}
+        issueDate={itemInfo.item.issueDate}
+        expirationDate={itemInfo.item.expirationDate}
+        credentialStatus={itemInfo.item.credentialStatus}
+        properties={[]}
+      />
+    );
+
+    return this.props.activeUser.identifiers.some((identifier: IUserIdentifier) => identifier.did === itemInfo.item.issuer.name)
```
Comment (nklomp) [inline src/screens/SSICredentialsOverviewScreen/index.tsx:94]: We should also check for the credential type. Current code basically means that you cannot delete any credential that is issued by the user. In the future we might add other self-asserted credential use cases, not being the wallet identity credential. So filtering on that makes sense

**Review by nklomp — CHANGES_REQUESTED** [review]
We should also check the credential type

File: src/screens/SSICredentialsOverviewScreen/index.tsx (hunk 1 above)
Comment (BtencateSphereon) [reply src/screens/SSICredentialsOverviewScreen/index.tsx]: added a check for the type/title of the vc

**Comment by nklomp** [comment]
LGTM

**Review by nklomp — APPROVED** [review]

**Pull request merged**